.editorconfig

```diff
@@ -5,6 +5,9 @@ charset = utf-8
 max_line_length = 80
 trim_trailing_whitespace = true
 
+[*.md]
+trim_trailing_whitespace = false
```

Comment:
@nate-double-u ,
I am guessing that this an override of line 6?
Does this enable a single space just before the carriage return?

Reply:
> I am guessing that this an override of line 6?

That's correct, this is an override only for `*.md` files, all others will still have `trim_trailing_whitespace` turned on.  Also of note, if there are other markdown file extensions being used in the site, this override will not cover them, and should probably be added (I did a cursory check but didn't see any).

> Does this enable a single space just before the carriage return?

Yes it does.

Reply:
OK. There was a former PR that cleaned up spaces at the end of lines, PR #16787.
I would assume that this setting would add the space in the right place 😄 .

Reply:
Primarily what I'm trying to accomplish with this PR is to stop the editorConfig plugins from automatically stripping all whitespace from the ends of lines in `.md` files on save — I've not gone through and reformatted anything, or tried to correct any existing lines.

What triggered this was some confusion working on my last PR. I kept having to add back spaces after saving, but before trying to make a commit, but every time I saved it would strip them out again. Had to disable plugins one at a time to figure out that it was editorconfig that didn't like the whitespaces.

Reply:
(I wish there was a https://gitignore.io for .editorconfig files, would make it easier to set them up)